### ISSUE TITLE
Authorization header has bytes type marker

### ISSUE THREAD

**Issue by jasonprado** [prschmid/zoomus#211]
I'm getting an `Invalid access token` error with zoomus 1.1.6 on Python 3.8.10 running in a Debian VM in a docker container based on `mcr.microsoft.com/vscode/devcontainers/python:3.8`.

When I print out the headers in `util.get_request`, it looks like this:
```
{'Authorization': "Bearer b'XXXXXXX'"}
```

The Python string annotation `b''` is showing up.

If I edit `util.generate_jwt` to look like this it's fixed:
```
token = jwt.encode(payload, secret, algorithm="HS256", headers=header).decode('utf-8')
```

Is there a version mismatch here? For some reason `jwt.encode` is producing `bytes` for me.

**Comment by jasonprado** [comment]
I suspect this is because I also have `twilio==6.59.0` installed which is requiring `PyJWT==1.7.1` (pinned). Maybe the fix here is for zoomus to require `PyJWT>=2.0` or something.

**Comment by jasonprado** [comment]
And if anyone finds themselves in a similarly annoying situation, here's a monkeypatch that's worked for me:
```
def new_generate_jwt(key, secret):
  import jwt
  import time
  header = {"alg": "HS256", "typ": "JWT"}
  payload = {"iss": key, "exp": int(time.time() + 3600)}
  return jwt.encode(payload, secret, algorithm="HS256", headers=header).decode('utf-8')
zoomus.util.generate_jwt = new_generate_jwt
```

**Comment by prschmid** [comment]
@jasonprado Indeed the issue you are running in to is due to the version of PyJWT. It's the same issue that was brought up in #157,  #182, and #171. We updated PyJWT in bd79bef4474db7486b36138248876dcc3741683b and was released in 1.1.4 (3cd5d13383c59c3aa7d05162edbbd072775a449a)
